### DISCUSSION TITLE
2021适配

### DISCUSSION THREAD

**Discussion by HernandoR** [xjtu-blacksmith/easymcm#7] [Q&A]
现有模板「2021.1.24」是否与2021年MCM/ICM适配？

**Comment by xjtu-blacksmith** [comment]
请检查 5.1 版本的[发布信息](https://github.com/xjtu-blacksmith/easymcm/releases/tag/v5.1)。美赛举办方历来只对论文首页（summary sheet，通过 Word 格式的模板给出）和部分细节（总页数、字体大小）等做出规定，根据今年发布的 2021 届首页模板，相关要求与去年无异，**最新版本 v5.1 模板应该仍然适配今年的比赛**。

> 你所说的 「2021.1.24」是本仓库最近一次 commit 的时间，自 5.1 版本以来仓库之中仅有 README 文件变动过。
>
> 我也还在与今年参赛的同学联系，请他们关注今年比赛动向。如因临时调整而出现模板不适配的情况，我会及时更新。
